src/Libraries/Watch3DNodeModels/Watch3D.cs

```diff
@@ -104,7 +104,7 @@ public class Watch3D : NodeModel
         // to be applied when the view model is constructed.
         internal XmlNode initialCameraData;
 
-        internal event Action<XmlNode> Deserialized;
+        public event Action<XmlNode> Deserialized;
```

Comment:
Why make public?

Reply:
to be able to unsubscribe from it later, but we should be able to use internal, I just need to use the internalsVisible attr